akka-actor-tests/src/test/scala/akka/io/dns/DnsSettingsSpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.io.dns
+
+import java.net.InetAddress
+
+import akka.actor.ExtendedActorSystem
+import akka.testkit.AkkaSpec
+import com.typesafe.config.ConfigFactory
+
+class DnsSettingsSpec extends AkkaSpec {
+
+  val eas = system.asInstanceOf[ExtendedActorSystem]
+
+  "DNS settings" must {
+
+    "use host servers if set to default" in {
+      val dnsSettings = new DnsSettings(eas, ConfigFactory.parseString(
+        """
+          nameservers = "default"
+          resolve-timeout = 1s
+        """))
+
+      dnsSettings.NameServers
```

Comment:
No assertion?

Reply:
Yeah i'll put a comment, it could be anything as it the OSes name servers, i just dont want it to throw